client/apps/edit/components/content/article_layouts/test/video.test.js

```diff
@@ -69,4 +69,13 @@ describe('EditVideo', () => {
     component.find(EditVideoPublished).simulate('click')
     expect(props.onChange.mock.calls[0][1].published).toBe(true)
   })
+
+  it('sets release_date on media', () => {
+    const component = mount(
+      <EditVideo {...props} />
+    )
+    const input = component.find('input').at(3)
+    input.simulate('change', { target: { value: '2017-02-07' } })
```

Comment:
Minor, but i believe finding input[type=date] would work here to give more specificity, in case the order of inputs changes in the UI.

Reply:
True true, nice!